src/plugins/initiatives/declaration.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+
+import deepFreeze from "deep-freeze";
+import type {PluginDeclaration} from "../../analysis/pluginDeclaration";
+import type {NodeType, EdgeType} from "../../analysis/types";
+import {NodeAddress, EdgeAddress} from "../../core/graph";
+
+export const nodePrefix = NodeAddress.fromParts(["sourcecred", "initiatives"]);
+export const edgePrefix = EdgeAddress.fromParts(["sourcecred", "initiatives"]);
+
+export const initiativeNodeType: NodeType = deepFreeze({
+  name: "Initiative",
+  pluralName: "Initiatives",
+  prefix: NodeAddress.append(nodePrefix, "initiative"),
+  defaultWeight: 1,
+  description:
+    "An initiative, formatted as a Discourse wiki topic using a fixed template.",
+});
+
+/*
+	Note on the forward and backward naming convention.
+	It follows the core/graph.js documentation to use
+	a <subject> <verb> <object> format to figure out
+	the directionality.
+*/
+
+/**
+ * A Discourse topic (src) TRACKS (verb) an initiative (dst).
+ * Forward: separate in terms of cred flow for now
+ * Backward: separate in terms of cred flow for now
+ */
+export const discourseTopicTracksInitiativeEdgeType: EdgeType = deepFreeze({
+  forwardName: "tracks",
+  backwardName: "is tracked by",
+  prefix: EdgeAddress.append(edgePrefix, "discourseTopicTracksInitiative"),
+  defaultWeight: {forwards: 0, backwards: 0},
+  description: "Connects an initiative to the Discourse topic that tracks it.",
+});
+
+/**
+ * An initiative (src) DEPENDS ON (verb) a dependency (dst).
+ * Forward: depending on something flows significant cred to the dependency.
+ * Backward: having a dependency does not flow much cred to the iniative,
+ * though does flow some to incentivize reuse and attribution.
+ */
+export const dependsOnEdgeType: EdgeType = deepFreeze({
+  forwardName: "depends on",
+  backwardName: "is a dependency for",
+  prefix: EdgeAddress.append(edgePrefix, "dependsOn"),
+  defaultWeight: {forwards: 1, backwards: 1 / 16},
+  description: "Connects an initiative to it's dependencies.",
+});
+
+/**
+ * An initiative (src) REFERENCES (verb) a reference (dst).
+ * Forward: reference for an initiative flows medium cred to the reference.
+ * Backward: having reference material does not flow much cred to the iniative,
+ * though does flow some to incentivize using existing research and attribution.
+ */
+export const referencesEdgeType: EdgeType = deepFreeze({
+  forwardName: "references",
+  backwardName: "is referenced for",
+  prefix: EdgeAddress.append(edgePrefix, "references"),
+  defaultWeight: {forwards: 1 / 2, backwards: 1 / 16},
+  description: "Connects an initiative to it's references.",
+});
+
+/**
+ * A contribution (src) CONTRIBUTES TO (verb) an initiative (dst).
+ * Forward: receiving a contribution flows a medium amount of cred to the iniative.
```

Comment:
nit: i think this should be phrased with the contribution as the subject, e.g. "a contribution flows a medium amount of cred to the initiative"

Reply:
Done! :+1: